stable/influxdb/values.yaml

```diff
@@ -164,6 +164,7 @@ config:
     compact_full_write_cold_duration: 4h0m0s
     max_series_per_database: 1000000
     max_values_per_tag: 100000
+    index_version: inmem
```

Comment:
Thanks, good job. But there needs another smaller change for the content of the line number 149   https://github.com/helm/charts/blob/fa6f96385e59207d90c9c20dae29c053fae75591/stable/influxdb/values.yaml#L149 , replace the doc address to (https://docs.influxdata.com/influxdb/v1.7/administration/config).

Reply:
Doc address changed